trade-finance-manager-ui/server/controllers/helpers/deal-cancellation-enabled.helper.ts

```diff
@@ -0,0 +1,13 @@
+import { DEAL_SUBMISSION_TYPE, DealSubmissionType, isTfmDealCancellationFeatureFlagEnabled, TEAM_IDS } from '@ukef/dtfs2-common';
+import { userIsInTeam } from '../../helpers/user';
+import { TfmSessionUser } from '../../types/tfm-session-user';
+
+export const dealCancellationEnabled = (submissionType: DealSubmissionType, user: TfmSessionUser): boolean => {
+  // TODO: DTFS2-7298: also check that the deal hasn't already been cancelled.
+
+  const userAllowedToCancelDeal = userIsInTeam(user, [TEAM_IDS.PIM]);
+  const dealCancellationFeatureFlagEnabled = isTfmDealCancellationFeatureFlagEnabled();
+  const acceptableSubmissionType = submissionType === DEAL_SUBMISSION_TYPE.AIN || submissionType === DEAL_SUBMISSION_TYPE.MIN;
+
+  return userAllowedToCancelDeal && dealCancellationFeatureFlagEnabled && acceptableSubmissionType;
```

Comment:
can this be a const please?

Reply:
I'm not sure there's any need to allocate the memory to reference this variable as it's the return value, and so it's clear it's returning a boolean which is referring to `isDealCancellationEnabled`. I think it might just be an extra line of code which is not particularly needed? I also think it's good to have how the boolean is calculated in easy access right at the end of the function. 

What do you think? Is there a reason why you'd want it to be a const?

Reply:
it looks better rather than many &&s.  However to keep it consistent, these can all be changed later